models/Enquiry.js

```diff
@@ -0,0 +1,23 @@
+var keystone = require('keystone');
+var Types = keystone.Field.Types;
+
+var Enquiry = new keystone.List('Enquiry', {
```

Comment:
Maybe call this `UpdateHandlerTest` since that's what it's for? we'll need to add field types (e.g. File) that wouldn't naturally exist in an enquiry